src/components/header/search.jsx

```diff
@@ -16,12 +19,14 @@ const transformItems = (items) =>
 export default function Search() {
 	return (
 		<div class={style.search}>
-			<DocSearch
-				apiKey={config.docsearch.apiKey}
-				indexName={config.docsearch.indexName}
-				appId={config.docsearch.appId}
-				transformItems={transformItems}
-			/>
+			<Suspense fallback={<p>Search</p>}>
```

Comment:
Will already be using `/compat` as their search is a React component, I don't think a spoofed suspense boundary would do much good here?

As for the fallback, shouldn't ever be shown to my knowledge in prod so probably doesn't matter. Just went with text to avoid too much shifting in dev & throttled connections more than anything.

Reply:
Could use ErrorBounday from preact-iso as well if you prefer